dist/c89-compatible/Hacl_HMAC.h

```diff
@@ -71,5 +72,37 @@ Hacl_HMAC_compute_sha2_512(
   uint32_t data_len
 );
 
+typedef struct K____uint32_t__uint64_t_s
```

Comment:
can you do a quick investigation and see why this appears here, and whether it's a kremlin bug, or that some other file needs it? (use `krml -d reachability`)

Reply:
After investigation, this structure is used in `Hacl_HMAC_compute_blake2b_32` : see [here](https://github.com/project-everest/hacl-star/blob/904acaf6a11492513c26be0da2172eab780b5c2c/dist/gcc-compatible/Hacl_HMAC.c#L646) (and there is a similar problem for blake2s...)
It seems that the problem comes from [this line](https://github.com/project-everest/hacl-star/blob/904acaf6a11492513c26be0da2172eab780b5c2c/code/hmac/Hacl.HMAC.fst#L291), and where `alloca` is actually [Hacl.Hash.Core.Blake2.alloca_blake2b_32](https://github.com/project-everest/hacl-star/blob/904acaf6a11492513c26be0da2172eab780b5c2c/code/hash/Hacl.Hash.Core.Blake2.fst#L82)